necst/rx/ccd.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from neclib.devices import CcdController as CCD_Device
+from necst_msgs.msg import CCDMsg
+
+from ... import config, namespace, topic
+from ..core import DeviceNode
```

Comment:
次にimportが余分だとされています。
testのエラーを見てもらえればいいですが、このNodeのコード内では確かにtimeとconfigが使われていないので削除しておいてください。
```suggestion
from neclib.devices import CcdController as CCD_Device
from necst_msgs.msg import CCDMsg

from ... import namespace, topic
from ..core import DeviceNode
```